play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -626,6 +647,44 @@ class OptionsPanel extends PSRoomPanel {
 					name="noselfhighlight" checked={PS.prefs.noselfhighlight || false} type="checkbox" onChange={this.handleOnChange}
 				/> Do not highlight when your name is said in chat</label>
 			</p>
+			<p>
+				<label class="checkbox"><input
+					name="leavePopupRoom" checked={PS.prefs.leavePopupRoom || false} type="checkbox" onChange={this.handleOnChange}
+				/> Confirm before leaving a room</label>
+			</p>
+			<p>
+				<label class="checkbox"><input
+					name="refreshprompt" checked={PS.prefs.refreshprompt || false} type="checkbox" onChange={this.handleOnChange}
+				/> Confirm before refreshing</label>
+			</p>
+			<p>
+				<label class="optlabel">
+					Language: {}
+					<select name="language" onChange={this.handleOnChange} class="button">
+						<option value="german" selected={PS.prefs.language === "german"}>Deutsch</option>
+						<option value="english" selected={PS.prefs.language === "english"}>English</option>
+						<option value="spanish" selected={PS.prefs.language === "spanish"}>Español</option>
+						<option value="french" selected={PS.prefs.language === "french"}>Français</option>
+						<option value="italian" selected={PS.prefs.language === "italian"}>Italiano</option>
+						<option value="dutch" selected={PS.prefs.language === "dutch"}>Nederlands</option>
+						<option value="portuguese" selected={PS.prefs.language === "portuguese"}>Português</option>
+						<option value="turkish" selected={PS.prefs.language === "turkish"}>Türkçe</option>
+						<option value="hindi" selected={PS.prefs.language === "hindi"}>हिंदी</option>
+						<option value="japanese" selected={PS.prefs.language === "japanese"}>日本語</option>
+						<option value="simplifiedchinese" selected={PS.prefs.language === "simplifiedchinese"}>简体中文</option>
+						<option value="traditionalchinese" selected={PS.prefs.language === "traditionalchinese"}>中文</option>
+					</select>
+				</label>
+			</p>
+			<p>
+				<label class="optlabel">
+					Tournaments: <select name="tournaments" class="button" onChange={this.handleOnChange}>
+						<option value="notify" selected={PS.prefs.tournaments === "notify"}>Notifications</option>
+						<option selected={!PS.prefs.tournaments}>No notifications</option>
```

Comment:
```suggestion
						<option value="" selected={!PS.prefs.tournaments}>No notifications</option>
						<option value="notify" selected={PS.prefs.tournaments === "notify"}>Notifications</option>
```

Probably the default should be the top option, for clarity.